src/styles/_mixins.js

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/react'
+import {
+  $smMobileBreakpoint,
+  $lgMobileBreakpoint,
+  $tabletBreakpoint,
+  $desktopBreakpointMinus,
+  $desktopBreakpoint,
+  $desktopBreakpointPlus,
+  $mediumDesktopBreakpoint,
+  $lgDesktopBreakpoint,
+} from './_variables'
+
+export const transition = ($args) => css`
+  -webkit-transition: ${$args};
+  -moz-transition: ${$args};
+  -ms-transition: ${$args};
+  -o-transition: ${$args};
+  transition: ${$args};
+`
+
+export const tablet = (content) => css`
+  @media (min-width: ${$tabletBreakpoint}) {
+    ${content}
+  }
+`
+
+export const desktop = (content) => css`
+  @media (min-width: ${$desktopBreakpoint}) {
+    ${content}
+  }
+`
+
+export const desktopBreakpointPlus = (content) => css`
```

Comment:
clean this code based on 3 break points  which names are in the other comment box.